avm/res/fabric/capacity/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,54 @@
+targetScope = 'subscription'
+
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+// e.g., for a module 'network/private-endpoint' you could use 'dep-dev-network.privateendpoints-${serviceShort}-rg'
```

Comment:
```suggestion
```

Reply:
Fixed.

Reply:
Probably a missing push, I cannot see comments removed from this file

Reply:
Pushed